cmd/backup/config.go

```diff
@@ -47,6 +47,7 @@ type Config struct {
 	BackupExcludeRegexp           RegexpDecoder   `split_words:"true"`
 	BackupSkipBackendsFromPrune   []string        `split_words:"true"`
 	GpgPassphrase                 string          `split_words:"true"`
+	GpgPublicKeys                 string          `split_words:"true"`
```

Comment:
I'm a bit torn on the plural name. Other keys in the config that use a plural s are comma separated list. so on first read I expected this to be a comma separated list as well, which it isn't.

Maybe this could be called `GpgPublicKeyRing` or `GpgArmoredKeyRing`, picking up the terminology from the gpg package?

Reply:
Yes, it can be a bit misleading and does not fit well into this project's terminology. I think using `GpgPublicKeyRing` is best, so its clear public keys are expected.

Reply:
fixed in [c76c13d](https://github.com/offen/docker-volume-backup/pull/456/commits/c76c13d02ee28226e926aba8e8c1b7101187a584)